### PR TITLE
nova and model fixes

### LINKED ISSUE #57: Review Nova resources
- contact_responses migration not working

### PR DESCRIPTION
**Pull request by arwaawan3**
Closes #57

### CODE REVIEW

File: src/Nova/Contact.php [hunk 1]
```diff
@@ -64,14 +73,14 @@ public function fields(Request $request)
                 FormType::PARTIES => 'Parties',
                 FormType::GROUPS => 'Groups',
                 FormType::EMPLOYMENT => 'Employment',
-            ])->required()->hideWhenUpdating(),
+            ])->rules('required')->hideWhenUpdating(),
```
Comment (wolfrednicolas) [inline src/Nova/Contact.php:76]: with this approch the validation is taken, with the `required()` not

Comment (joshtorres) [reply src/Nova/Contact.php:76]: Sorry I don't understand this comment

**Review by joshtorres — APPROVED** [review]
Looks good. Thank you!

File: src/Nova/Contact.php (hunk 1 above)
Comment (wolfrednicolas) [reply src/Nova/Contact.php:76]: @joshtorres  with `->required()` doesnt show the validation properly... with rules('required') does

**Pull request merged**